source/assets/javascripts/components/Leaderboards/ElevationLeaderboard/index.js

```diff
@@ -49,6 +48,10 @@ export default React.createClass({
 
     data = filterTeams(this.props.teamPageIds, data)
 
+    data.forEach(function(item, index) {
+      item.rank = index + 1
+    })
+
```

Comment:
Could do. Don't mind. I tend to go for breaking different actions apart so it's easier to read. 
– This happens
– Then this

I always find chaining to much a little confusing. But that just my preference.